### ISSUE TITLE
What about NOT using Third Party Tools?

### ISSUE THREAD

**Issue by FC-EricWeller** [MicrosoftDocs/OfficeDocs-Exchange#108]
This article's title is about migrating mailboxes from one tenant to another.  It offers a scenario "using third party tool" but no scenario or link to *not* using a third party tool.  The implication seems to be that there is not built-in migration process?  The article should state so plainly, or provide links to any built-in processes that do exist.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 59062d31-9618-f316-c6b9-1ed30319aa7f
* Version Independent ID: b5d5266d-ad7d-bc44-53a3-59aecf46391e
* Content: [How to migrate mailboxes from one Office 365 tenant to another](https://docs.microsoft.com/en-us/exchange/mailbox-migration/migrate-mailboxes-across-tenants)
* Content Source: [Exchange/ExchangeOnline/mailbox-migration/migrate-mailboxes-across-tenants.md](https://github.com/MicrosoftDocs/OfficeDocs-Exchange/blob/public/Exchange/ExchangeOnline/mailbox-migration/migrate-mailboxes-across-tenants.md)
* Service: **exchange-online**
* Product: **exchange**
* GitHub Login: @msdmaguire
* Microsoft Alias: **dmaguire**

**Comment by JBines** [comment]
The option for Off boarding to On-Prem and then On-Boarding could be added.

**Comment by AndreaBarr** [comment]
@FC-EricWeller Thank you for submitting this feedback.

Thank you for pointing this out to us. We will get the migrating mailbox area updated. 

I was able to get this into the backlog for the SharePoint team. The team will get this prioritized into a sprint plan. Thank you for reporting and making the docs better. Much appreciated.

I made a note to request the team to update this when the work is complete.
